main.py

```diff
@@ -75,6 +75,20 @@ def main():
         help="Browser to use for cookies.",
     )
 
+    parser.add_argument(
+        "--cookiefile",
+        type=str,
+        default=None,
+        help="Path of netscaped cookie file.",
+    )
+
+    parser.add_argument(
```

Comment:
This shouldnt be an option. Shorts dont need to be considered.

Reply:
There is feat request so added it as a bonus

Reply:
Not needed, you can remove the option and just skip shorts to make the script faster

Reply:
Default is false so, it doesn't effect speed of execution unless explicitly said.
But still if you hate it we can remove it

Reply:
No I am not saying that speed will differ, I am saying to skip shorts, the option itself has no usecase, there's no scenario where one would want to watch shorts